### DISCUSSION TITLE
Will this be updated to support 4090 launch?

### DISCUSSION THREAD

**Discussion by RowanUnderwood** [jef/streetmerchant#3047] [Ideas and feature requests]
Will this be updated to support 4090 launch?  Hoping for the best, but prepared for the scalpers to get 'em all again :*[  Thanks for helping me get a PS5 and 3090 at MSRP though :D!

**Comment by oriddlero** [comment]
Guess not.  Fail sauce.

**Comment by jef** [comment]
Great question! I don't think I'll be making time to add this in, but others are also looking for this addition.

https://github.com/jef/streetmerchant/issues/3053

**Reply by jef** [reply]
Resolved with https://github.com/jef/streetmerchant/pull/3085

**Comment by oriddlero** [comment]
Damn, I see you just did a release too :(  4090FE is a real PITA to score.  I see you added many other GPUs why not the 4090FE?

**Reply by jef** [reply]
Hey @oriddlero, the 4090 FE is now supported. Thanks!
